pom.xml

```diff
@@ -1006,8 +1006,18 @@
                         <groupId>com.fasterxml.jackson.core</groupId>
                         <artifactId>jackson-annotations</artifactId>
                     </exclusion>
+                    <exclusion>
+                        <groupId>org.eclipse.jetty.http2</groupId>
+                        <artifactId>http2-common</artifactId>
+                    </exclusion>
                 </exclusions>
             </dependency>
+            <!-- Override http2-client version coming from solr-->
+            <dependency>
+                <groupId>org.eclipse.jetty.http2</groupId>
+                <artifactId>http2-common</artifactId>
+                <version>11.0.25</version>
```

Comment:
Should we use a property value instead?

Reply:
I don't think it's necessary. This is how we usually do it for transitive dependency which we don't actively use in framework.